plugin.video.retrospect/addon.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon  id="plugin.video.retrospect"
+        version="5.0.0.0~beta2"
```

Comment:
no beta (alpha or rc) in the official Kodi repository

Reply:
I have removed the external libs and created add-ons for them (https://github.com/xbmc/repo-scripts/pull/1089 and https://github.com/xbmc/repo-scripts/pull/1090). Also reverted to the standard urllib3